.env.example

```
@@ -30,3 +33,10 @@ DB_PASS=secret
 DB_NAME=postgres
 DB_PORT=5432
 DB_HOST=localhost
+
+# Mail Configuration
+SMTP_HOST=127.0.0.1
+SMTP_PORT=1025
+SMTP_USER=null
+SMTP_PASS=null
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SMTP_PASS key should go before the SMTP_PORT key

(UnorderedKey)